README.md

```diff
@@ -38,6 +40,27 @@ Places the following files in the destination:
   {"source_ami": "ami-5731123e"}
   ```
 
+### `out`: Check that an AMI exists
+
+For bookkeeping in Concourse. Expects a JSON file at `input/json` with an object containing an `ImageId` property e.g.
```

Comment:
```suggestion
For bookkeeping in Concourse—generates a new version immediately so it can be passed to the next job in a pipeline without needing a `check` to run. Expects a JSON file at `input/json` with an object containing an `ImageId` property e.g.
```